### DISCUSSION TITLE
Switching from LayerStrip and HeightStrip to storing colors in .json

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3826] [Ideas]
Currently the colors for baking a chart are stored in images within a bundle. It might be a lot easier for everyone if these were just stored in a RGB or Hex format within a .json file.

EDIT: Instead of a JSON file, the best route would be using the LevelAsset.

**Comment by SDGNelson** [comment]
Yeah, they are an ideal candidate for rewriting/reworking.
